pkg/pmem-device-manager/pmd-ndctl.go

```diff
@@ -14,9 +14,16 @@ import (
 )
 
 const (
-	// 1 GB align in ndctl creation request has proven to be reliable.
-	// Newer kernels may allow smaller alignment but we do not want to introduce kernel dependency.
-	ndctlAlign uint64 = 1024 * 1024 * 1024
+	ndctlAlign uint64 = 2 * 1024 * 1024
```

Comment:
Document what this alignment is (just a fallback, right?).

Reply:
this goes away, I removed passing Align from device manager to region.go, means it will be unused zero field